website/package.json

```diff
@@ -19,10 +19,10 @@
     },
     "dependencies": {
         "@astrojs/node": "^8.3.4",
-        "@auth/core": "^0.37.4",
```

Comment:
Let's please not downgrade this... I'd rather accept some "lint ignore" comments.

Reply:
I tried to add ignore statements to get the tests to pass for about 2hours and then gave up - the linter really wants to check the directory, potentially we could add more patches. If you have more experience with this we can maybe try together in a call.

Reply:
Thanks so much for the fix!!!